glide.lock

```diff
@@ -283,7 +216,7 @@ imports:
 - name: gopkg.in/natefinch/lumberjack.v2
   version: 20b71e5b60d756d3d2f80def009790325acc2b23
 - name: gopkg.in/yaml.v2
-  version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
+  version: a5b47d31c556af34a302ce5d659e6fea44d90de0
```

Comment:
These lines in particular aren't for gengo, but given the amount of difficulties in investigating dependency issues perhaps these dependencies should be checked more closely? As far as gengo goes, the version is an upgrade for master to a commit from a few days ago.